### PR TITLE
Update code.md

### PR DESCRIPTION
**Pull request by Jayita10**

### CODE REVIEW

File: code.md [hunk 1]
```diff
@@ -4,4 +4,6 @@
 - https://github.com/howdyai/botkit-cms : An open tool for designing, building and managing interactive dialog systems
```
Comment (dermatologist) [inline code.md:6]: Hi @Jayita10 

The task is to add a header to the markdown file as below:

#Code

Thanks

**Review by dermatologist — CHANGES_REQUESTED** [review]

File: code.md (hunk 1 above)
Comment (Jayita10) [reply code.md:6]: Sorry I didn't understand. Please Check now

Comment (dermatologist) [reply code.md:6]: perfect! Thanks!
I see that you are into web development. Please have a look at https://github.com/dermatologist/fhir-questionnaire-render-react when you have time!

**Review by dermatologist — APPROVED** [review]

**Comment by dermatologist** [comment]
Thanks, @Jayita10 
I see that you are into web development. Please have a look at https://github.com/dermatologist/fhir-questionnaire-render-react when you have time!

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/dermatologist/fhir-questionnaire-render-react">dermatologist/fhir-questionnaire-render-react</a></strong></div><div>Render FHIR Questionnaire as a web form using FHIRFormJS - dermatologist/fhir-questionnaire-render-react</div></blockquote>

**Pull request merged**